### PR TITLE
Fix typos in `webrct_signalling/README.md`

### PR DESCRIPTION
**Pull request by bend-n**
`./networking/webrct_signalling/README.md` had some strange english problems.

### CODE REVIEW

File: networking/webrtc_signaling/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Messages with payload (used to transfer WebRTC parameters) are:
 - `A: <ID>`, used to send an answer.
 - `C: <ID>`, used to send a candidate.
 
-When sending the parameter, a client will set `<ID>` as the destination peer, the server will replace it with the id of the sending peer, and rely it to the proper destination.
+When sending the parameter, a client will set `<ID>` as the destination peer, and the server will replace it with the id of the sending peer, and relay it to the proper destination.
```
Comment (aaronfranke) [inline networking/webrtc_signaling/README.md:40]:
```suggestion
When sending the parameter, a client will set `<ID>` as the destination peer, the server will replace it with the id of the sending peer, and relay it to the proper destination.
```

**Comment by bend-n** [comment]
what

**Comment by bend-n** [comment]
How do I undo that

**Comment by bend-n** [comment]
Ah, shite, I ammended a shallow clone. Let me re-do this.

**Pull request closed without merging**